src/dialects/mssql/query-interface.js

```diff
@@ -84,6 +90,8 @@ export class MsSqlQueryInterface extends QueryInterface {
 
     const sql = this.queryGenerator.upsertQuery(tableName, insertValues, updateValues, where, model, options);
 
-    return await this.sequelize.query(sql, options);
+    delete options.replacements;
```

Comment:
`queryRaw` will throw if `options.replacements` is specified in case someone tries to use it like `query`
I delete it before calling queryRaw to prevent it from throwing. I thought I documented it each time but it's missing here